streams/us_local.m3u

```diff
@@ -406,6 +408,8 @@ https://cdn3.wowza.com/5/WDIrTW5sM1JEY1NN/napatv/G0360_004/playlist.m3u8
 https://livevideo.tegnadigital.com/king/v1/master/f9c1bf9ffd6ac86b6173a7c169ff6e3f4efbd693/KING/elvs/live.m3u8
 #EXTINF:-1 tvg-id="KUSADT1.us",NBC 9 Denver CO (KUSA-DT1) (1080p)
 https://livevideo.tegnadigital.com/kusa/v1/master/f9c1bf9ffd6ac86b6173a7c169ff6e3f4efbd693/KUSA/elvs/live.m3u8
+#EXTINF:-1 tvg-id="WJARDT1.us",NBC 10 Providence MA (1080p)
+https://pubads.g.doubleclick.net/ssai/event/kpannrCBQ5KwdV3Hag3Wog/master.m3u8
```

Comment:
also geo-blocked?

Reply:
I mean it is absolutely sure geo-blocked just like the link above from the same source

Reply:
Shouldn't be, it's a redirect to https://linear-694.frequency.stream/dist/newson/694/hls/master/playlist.m3u8

Reply:
Geo-blocked just for me then

Reply:
> Shouldn't be, it's a redirect to https://linear-694.frequency.stream/dist/newson/694/hls/master/playlist.m3u8

This one is working for me.